core/templates/domain/topics_and_skills_dashboard/TopicsAndSkillsDashboardFilterObjectFactorySpec.ts

```diff
@@ -16,11 +16,13 @@
  * @fileoverview Unit tests for TopicsAndSkillsDashboardFilterObjectFactory.
  */
 
-import {
-  TopicsAndSkillsDashboardFilter,
+import { TopicsAndSkillsDashboardFilter,
```

Comment:
Does breaking after { not work? It's more consistent with the existing codebase style, I think.

Reply:
Done.